x/ibc/07-tendermint/misbehaviour.go

```diff
@@ -69,21 +68,21 @@ func checkMisbehaviour(
 		)
 	}
 
-	// Evidence is within the trusting period. ValidatorSet must have 2/3 similarity with trusted FromValidatorSet
-	// check that the validator sets on both headers are valid given the last trusted validatorset
-	// less than or equal to evidence height
+	// - Evidence is within the trusting period.
```

Comment:
Doesn't look like this check is implemented

Reply:
Yes, this should be checked, see https://github.com/cosmos/ics/pull/367